packages/fluentui/react-northstar/test/specs/components/Popup/Popup-test.tsx

```diff
@@ -243,8 +243,9 @@ describe('Popup', () => {
       expect(document.querySelector(`#${contentId}`)).toBe(null);
       expect(document.querySelector(`#${contentId2}`)).toBeDefined();
 
-      ReactDOM.unmountComponentAtNode(attachTo);
-      document.body.removeChild(attachTo);
+      // Error: Uncaught [NotFoundError: The node to be removed is not a child of this node.]
+      // ReactDOM.unmountComponentAtNode(attachTo);
```

Comment:
TODO: figure out what's causing this issue

Reply:
Still not sure what caused this but wrapping it in `ReactTestUtils.act` fixed it ([c76c788](https://github.com/microsoft/fluentui/pull/24356/commits/c76c788726c85327c86ba97c65da33d34edf402d))